docs/sources/set-up/install/windows.md

```diff
@@ -10,39 +10,40 @@ weight: 500
 
 # Install {{% param "FULL_PRODUCT_NAME" %}} on Windows
 
-You can install {{< param "PRODUCT_NAME" >}} on Windows as a standard graphical install, or as a silent install.
+You can install {{< param "PRODUCT_NAME" >}} on Windows as a standard graphical install, a WinGet install, or a silent install.
 
 ## Standard graphical install
 
 To do a standard graphical install of {{< param "PRODUCT_NAME" >}} on Windows, perform the following steps.
 
-1. Navigate to the [latest release][latest] on GitHub.
-
+1. Navigate to the [releases page][releases] on GitHub.
 1. Scroll down to the **Assets** section.
+1. Download and extract `alloy-installer-windows-amd64.exe.zip` or download `alloy-installer-windows-amd64.exe`.
+1. Double-click on `alloy-installer-windows-amd64.exe` to install {{< param "PRODUCT_NAME" >}}.
 
-1. Download the file called `alloy-installer-windows-amd64.exe.zip`.
+The installer places {{< param "PRODUCT_NAME" >}} in the default directory `%PROGRAMFILES%\GrafanaLabs\Alloy`.
 
-1. Extract the downloaded file.
+## WinGet install
 
-1. Double-click on `alloy-installer-windows-amd64.exe` to install {{< param "PRODUCT_NAME" >}}.
+To install {{< param "PRODUCT_NAME" >}} with WinGet, perform the following steps.
 
-{{< param "PRODUCT_NAME" >}} is installed into the default directory `%PROGRAMFILES%\GrafanaLabs\Alloy`.
+1. Make sure that the [WinGet package manager](https://learn.microsoft.com/en-gb/windows/package-manager/winget/) is available.
+1. Run the following command as Administrator in PowerShell.
```

Comment:
It is not necessary to run as an administrator, as the installer will elevate to request permission if required.

This can also be done in any terminal for Windows, it doesn't have to be from PowerShell.